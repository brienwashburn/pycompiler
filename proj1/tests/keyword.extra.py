class ;  			#this is a comment
def pass True \
False
