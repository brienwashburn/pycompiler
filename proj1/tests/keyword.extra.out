(KEYWORD class)
(PUNCT ";")
(NEWLINE)
(KEYWORD def)
(KEYWORD pass)
(KEYWORD True)
(KEYWORD False)
(NEWLINE)
(ENDMARKER)
